tuxemon/core/components/event/actions/player.py

```diff
@@ -253,42 +253,19 @@ def add_monster(self, game, action):
 
         game.player1.add_monster(current_monster)
 
-
-    def add_item(self, game, action):
-        """Adds an item to the current player's inventory. The action parameter must contain an
-        item name to look up in the item database.
-
-        :param game: The main game object that contains all the game's variables.
-        :param action: The action (tuple) retrieved from the database that contains the action's
+    def add_item(self,action):
```

Comment:
This method still needs the `game` argument. All event methods take `game, action` from the Event Engine defined in `core.components.event`.